test/Scripts.Integration.Test/SentryUpdateSetup.cs

```diff
@@ -0,0 +1,62 @@
+using UnityEditor;
+using UnityEditor.PackageManager.Requests;
+using UnityEditor.PackageManager;
+using UnityEngine;
+using System.Linq;
+
+[InitializeOnLoad]
+public static class Startup
+{
+    static AddRequest AddRequest;
+    static ListRequest ListRequest;
+
+    const string SentryPackageName = "io.sentry.unity";
```

Comment:
Seems to be an exact copy-paste of the other clas other than this package name. Please refactor this so we can just provide the package as an argument instead